ruby/private/bundle_fetch/gemfile_lock_parser.bzl

```diff
@@ -0,0 +1,191 @@
+"""
+Parses a Gemfile.lock purely in Starlark.
```

Comment:
did we also inherit some unit tests for this?

Reply:
No, there were no unit tests.